docs/concepts.rst

```diff
@@ -158,6 +158,96 @@ Example DAG with functional abstraction
 
 .. _concepts:dagruns:
 
+executor_config
+===============
+
+The executor_config is an argument placed into operators that allow airflow users to override tasks
+before launch. Currently this is primarily used by the KubernetesExecutor, but will soon be available
+for other overrides.
+
+pod_override
+------------
+
+When using the KubernetesExecutor, Airflow offers the ability to override system defaults on a per-task basis.
+To utilize this functionality, create a Kubernetes V1pod object and fill in your desired overrides.
```

Comment:
Can you use ``:class::`` role when you refer to class for the first time?

Reply:
*checks PST to poland time converter*

...

@mik-laj when do you sleep?...